dependencies.repos

```diff
@@ -4,8 +4,8 @@
     version: 0.2.1
 - git:
     local-name: noether
-    uri: https://github.com/marip8/noether.git
-    version: 3cff4cb87cbf45e21a760e6dd7907b9e604847cd
+    uri: https://github.com/marrts/noether.git
+    version: a33510e891d1a954b4851896354ffad03de6f3e2
```

Comment:
Is this something we can contribute back to the noether upstream? It would be nice to support a 22.04 build there

Reply:
Probably some stuff that'll work for 22.04, but ultimately there's stuff in Noether that is ROS specific and part of the difference in this branch is that it ignores all the ROS specific stuff. We can discuss more over on Noether too if you want to make an issue there.